examples/gallery/image_bright_regions_gallery_example.py

```diff
@@ -0,0 +1,95 @@
+
+# coding: utf-8
+
+"""
+====================================================
+Manipulating Map image data - finding bright regions
+====================================================
+
+This example shows how you can do basic image processing on SunPy map image data. In this example, we try to find the brightest regions in an AIA image, and count the approximate number of regions of interest.
+"""
+
+##############################################################################
+# First, import the modules we will need:
+
+
+import sunpy
+from sunpy import map
+import matplotlib.pyplot as plt
+import numpy as np
+from scipy import ndimage
+
+##############################################################################
+# Now, we create a SunPy Map object from an AIA FITS file.
+
+
+aiamap = map.Map('AIA20110607_062209_0193.fits')
+
+##############################################################################
+# Let's plot the map. Here we use the `clim` method to clip the colour table. This show features in an image with high dynamic range more clearly.
+
+
+aiamap.plot()
+plt.clim([0,2500])
+plt.colorbar()
+plt.show()
+
+##############################################################################
+# Now we want to find the brightest regions in this image. We start by finding the maximum value in the image data.
+
+
+m = np.max(aiamap.data)
+
+##############################################################################
+# Now we want to make a mask, which tells us which regions are bright. We choose the criterion that the data should be at least 5% of the maximum value. Pixels with intensity # values greater than this are included in the mask, while all other pixels are excluded. 
+
+
+mask = aiamap.data > m*0.05
+mask
+
+##############################################################################
+# Mask is a `boolean` array. It can be used to modify the original map data.
+
+
+
+data2 = aiamap.data * mask
+
+##############################################################################
+# We now have a new set of data, where all values outside the mask are set to zero. We can create a new SunPy map structure with this data.
+
+
+aiamap2 = map.Map(data2, aiamap.meta)
```

Comment:
Using the mask you could then plot it and the visualisation would done, no need to create another `map`.